src/status_im/contexts/wallet/wallet_connect/utils/data_store.cljs

```diff
@@ -73,3 +73,7 @@
   (-> db
       get-db-current-request-event
       get-request-params))
+
+(defn data-item->array
+  [data]
+  (map (fn [[k v]] [k v]) data))
```

Comment:
can simplify this by using `(mapv identity data)`, which will achieve the same thing, also removing the need for  the util.

Reply:
mapv returns a vector, whereas map returns a lazy sequence, and here to pass to flat-list i need lazy sequence

Reply:
you're right, then just `(map identity data)` will do